core/templates/dev/head/pages/exploration_editor/translation_tab/state_translation_directive.html

```diff
@@ -2,16 +2,24 @@
   <div class="tarnslation-nav md-tab">
```

Comment:
not in this PR but can you fix this, translation*

Reply:
Done!